README.md

```diff
@@ -7,130 +7,76 @@
-
-To build any of [examples](examples/), you need `cargo-make`. Install it by `cargo install cargo-make` (or see [cargo-make](https://github.com/sagiegurari/cargo-make#installation) for more).
-
-Run [examples](examples/) with `cargo make start example_name` from the Seed repository root. Here's our counter example:
-
-```rust
-use seed::{prelude::*, *};
-
-// `init` describes what should happen when your app started.
-fn init(_: Url, _: &mut impl Orders<Msg>) -> Model {
-    Model::default()
-}
-
-// `Model` describes our app state.
-type Model = i32;
-
-// `Msg` describes the different events you can modify state with.
-enum Msg {
-    Increment,
-}
-
-// `update` describes how to handle each `Msg`.
-fn update(msg: Msg, model: &mut Model, _: &mut impl Orders<Msg>) {
-    match msg {
-        Msg::Increment => *model += 1,
-    }
-}
-
-// `view` describes what to display.
-fn view(model: &Model) -> Node<Msg> {
-    div![
-        "This is a counter: ",
-        C!["counter"],
-        button![
-            model,
-            ev(Ev::Click, |_| Msg::Increment),
-        ],
-    ]
-}
-
-#[wasm_bindgen(start)]
-pub fn start() {
-    // Mount the `app` to the element with the `id` "app".
-    App::start("app", init, update, view);
-}
+Seed is a Rust front-end framework for creating fast and reliable web apps with an Elm-like architecture.
+
+- completely written in Rust, including the templating system (e.g. `div!` macro).
+- built-in state management that is based on the Elm architecture.
+- a batteries-included approach with a focus on developer experience.
+- clear and extensive documentation for Rust beginners and pros alike.
+- WebAssembly.
+
+## Why Seed?
+Seed allows you to develop the front-end with all the benefits of Rust, meaning speed, safety, and too many more things to count.
+
+The Seed templating system uses a macro syntax that makes Rustaceans feel right at home. This means linting, formatting, and commenting will work, and it's all in Rust. This is opposed to a JSX-like syntax that relies on IDE extensions to improve the developer experience.
+
+Seed has a batteries-included approach. This means less time writing boilerplate and less time installing dependencies.
+
+## Why not Seed?
+- It's under active development. That comes with occasional breaking changes.
+- WebAssembly is new. In the present time, interacting with the DOM isn't as performant as JavaScript.
+- The community is smaller. It's harder to find support outside of Discord.
+- Pre-built components are rare. You will likely have to roll your own components such as date pickers.
+- No server-side rendering yet [(#232)](#232).
+
+
+## Getting Started
+To get started right away, we can use quickstart template:
+```sh
+cargo install cargo-generate
+cargo install trunk
+cargo install wasm-bindgen-cli
+cargo generate --git https://github.com/seed-rs/seed-quickstart.git --name seed-quickstart
+cd seed-quickstart
+trunk serve
```

Comment:
I didn't know `seed-quickstart` works with Trunk. Did you test it?

Reply:
Yes, it's working. I'll add a note about linking version to wasm-bindgen.